h2o-bindings/bin/custom/R/gen_targetencoder.py

```diff
@@ -5,7 +5,24 @@
 parms$ignored_columns <- args$x_ignore
```

Comment:
adding backwards compatibility support for renamed params `k`, `f`, `noise_level`.